app/components/view_component/form/base_component.rb

```diff
@@ -44,7 +44,7 @@ def before_render
 
       def combine_options!
         @options = (self.class.default_options.deep_dup || {}).deep_merge(options).tap do |opts|
-          opts[:class] = class_names(options[:class], html_class) if (html_class || options[:class]).present?
+          # opts[:class] = class_names(options[:class], html_class) if (html_class || options[:class]).present?
```

Comment:
@Spone Not sure how to deal with this

Reply:
Let's copy https://github.com/rails/rails/blob/f95c0b7e96eb36bc3efc0c5beffbb9e84ea664e4/actionview/lib/action_view/helpers/tag_helper.rb#L329-L345 or https://github.com/komposable/klassnames/blob/master/lib/klassnames/classnames_helper.rb#L2-L19 to support Rails 6.0

Reply:
We will see in #21 then.